go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/vmware/cluster-api-provider-cloud-director
 
-go 1.20
```

Comment:
This is a go version update. Sometimes this needs a change to build scripts. Do we need this update or was this accidental?

Reply:
This came is a part of common core. Common core has been at go 1.21 for like 4 months now